l10n_br_fiscal/data/operation_data.xml

```diff
@@ -83,6 +84,36 @@
         <field name="state">approved</field>
     </record>
 
+    <record id="fo_venda_final" model="l10n_br_fiscal.operation">
+        <field name="code">Venda Consumo Final</field>
```

Comment:
```suggestion
        <field name="code">Venda Consumidor Final</field>
```

Reply:
I don't know if I agree with this change. The fiscal configuration of final consumption is broader than that. It can be a sale to a final consumer or for a company who will not sell the product again. In such case the product will be bought by a company, who will not consume the product, but make it available for its employees to consume the product. The sale will be for final consumption, but not for the final consumer per se.